components/widgets/title/index.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+const Title = styled.h1`
+  flex: 1 1 100%;
+  font-size: 60px;
+  margin: 0;
+  padding-bottom: 20px;
```

Comment:
See comment above.

Reply:
I removed the paddingBottom. This looks better, especially if you use multiple <Title /> in your dashboard.